src/components/Outcomes/OutcomeGenerator/OutcomeGenerator.tsx

```diff
@@ -211,16 +215,23 @@ export const OutcomeGenerator: FC<OutcomeGeneratorProps> = () => {
             title="XM Cloud Introduction GitHub Repo: Shows Next.js and .NET headless sites migrated from XM 10.2"
           />
         </ConditionalResponse>
-        <ConditionalResponse
-          condition={outcomeConditions.desiredFrameworks.nextjs}
-        >
+        <ConditionalResponse condition={outcomeConditions.desiredFrameworks.nextjs}>
           <LinkCard
             link="https://thetombomb.com/posts/nextjs-hosting-alternatives"
-          title="Beyond Vercel: Hosting Alternatives for Next.js"
+            title="Beyond Vercel: Hosting Alternatives for Next.js"
+          />
+        </ConditionalResponse>
+        <LinkCard
+          link="https://jackspektor.medium.com/estimating-sitecore-xp-to-xm-cloud-upgrade-what-challenges-lies-ahead-226d1c36b8e"
```

Comment:
I'm hesitant to link to medium.com articles. I couldn't find a clear answer but it seems a visitor who is not a member can only view 3 articles a month on medium. What's unclear is if being a member is just signing up or being a member is paying the $5 a month. 

Also the article can be set to be freely available or behind the paywall so that's another factor to consider.

Reply:
For most articles, the paywall is not an issue. Some articles are behind the paywall, but most are not. We should definitely be checking if a Medium article we are linking to is behind the paywall. I have a Medium account that I don't use, and I am not able to control whether it is behind the paywall or not. Apparently you have to be in the Partner program for that.

* [Check paywall status](https://help.medium.com/hc/en-us/articles/5144837010839-How-can-I-check-if-my-story-is-behind-the-paywall-)
* [Remove from paywall](https://help.medium.com/hc/en-us/articles/360018637634-Remove-story-from-the-paywall)

From what I can tell, none of Jack Spektor's articles on Medium are paywalled.

Reply:
In that case I'll approve the PR.